tests/integration/test_storage_iceberg/test.py

```diff
@@ -3161,7 +3161,125 @@ def test_compressed_metadata(started_cluster, storage_type):
         f"/iceberg_data/default/{TABLE_NAME}/",
         f"/iceberg_data/default/{TABLE_NAME}/",
     )
-
+    
     create_iceberg_table(storage_type, instance, TABLE_NAME, started_cluster, explicit_metadata_path="")
 
     assert instance.query(f"SELECT * FROM {TABLE_NAME} WHERE not ignore(*)") == "1\tAlice\n2\tBob\n"
+
+
+@pytest.mark.parametrize("format_version", ["1", "2"])
+@pytest.mark.parametrize("storage_type", ["s3", "azure", "local"])
+def test_writes(started_cluster, format_version, storage_type):
```

Comment:
For all tests for writes we need to verify that Spark is able to read written data too.